app/uk/gov/hmrc/minorentityidentificationfrontend/services/StorageService.scala

```diff
@@ -110,20 +110,20 @@ class StorageService @Inject()(connector: StorageConnector) {
         case None => Json.obj()
       }
 
-      val identifiersMatchString: Boolean =  optIdentifiersMatch match {
-        case Some(SuccessfulMatch) => true
-        case _ => false
+      val identifiersMatchBlock: JsObject = optIdentifiersMatch match {
+        case Some(aKnownFactsMatchingResult) => Json.obj("identifiersMatch" -> aKnownFactsMatchingResult.equals(SuccessfulMatch))
+        case None                            => Json.obj()
```

Comment:
This needs to cater for the scenario where there is no identifiersMatch stored (currently a few of our flows do this) because it's breaking the ATs and current flows we have. It still works if done how it is on the left here

Reply:
Yes. At the moment for the Overseas journey, Unincorporated Associations and the Trust journey that is live it's set up so that it does not store any of the identifiersMatch, businessVerification or registrationStatus fields and then in the journey data retrieval method if the status is not there we send it back as identifiersMatch = false, businessVerification = unchallenged and registrationStatus = not called. This will probably be better if we update how we do it but it doesn't need to be done in this ticket. Also if we don't allow for the scenario where a user does not have an identifiersMatch value stored (and return false) it will break user currently in journeys in live potentially

Reply:
Okay ... sorted ... we have to return false in case identifierMatch is not stored (None) due to some legacy code.